Makefile

```diff
@@ -4,7 +4,7 @@ GITHUB_ORG="pactflow"
 PACTICIPANT := "pactflow-example-consumer"
 GITHUB_WEBHOOK_UUID := "04510dc1-7f0a-4ed2-997d-114bfa86f8ad"
 PACT_CHANGED_WEBHOOK_UUID := "8e49caaa-0498-4cc1-9368-325de0812c8a"
-PACT_CLI="docker run --rm -v ${PWD}:${PWD} -e PACT_BROKER_BASE_URL -e PACT_BROKER_TOKEN pactfoundation/pact-cli:latest"
+PACT_CLI="docker run --rm -v ${PWD}:${PWD} -e PACT_BROKER_BASE_URL -e PACT_BROKER_TOKEN -e PACT_BROKER_FEATURES=deployments pactfoundation/pact-cli:0.46.1.0"
 
```

Comment:
For users who are looking at this diff who came from the "Deployments and releases" documentation, please note that the `PACT_BROKER_FEATURES` environment variable is not required any more, and that you should definitely pin your pact-cli version, rather than using `latest`.